site/content/docs/5.2/examples/cards/index.html

```diff
@@ -15,72 +15,72 @@
 
   <div class="container-xxl pt-5">
     <div class="row mx-0">
-     <h1>Cards filed black</h1>
+     <h1>Cards filled black</h1>
```

Comment:
Has the arabic wording was generated by Google Translate? If so, it worths regenerating it.